packages/legacy/babel/static/plugins/inject-package-version.js

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+const t = require('@babel/types');
+
+const {version} = require('../../package.json');
+
+/**
+ * Simple babel transform for ensuring that every WebexPlugin (and WebexCore)
+ * have the correct version property
+ * @returns {Object}
+ */
+module.exports = function injectPackageVersion() {
+  return {
+    visitor: {
+      /**
+       * Adds a version property to all SparkPlugins
+       * @param {Object} path
+       * @returns {void}
+       */
+      CallExpression(path) {
+        if (t.isMemberExpression(path.get('callee'))) {
+          if (
+            path.node.callee.object.name === 'WebexPlugin' &&
+            path.node.callee.property.name === 'extend'
+          ) {
+            const def = path.node.arguments[0];
+            const visited = def.properties.reduce(
+              (acc, p) => acc || t.isObjectProperty(p, {key: 'version'}),
```

Comment:
Can we add comments on what it exactly does , little confused with the names

Reply:
This was copy-pasted from the root config, I do not have any idea what this does at the moment, but, it was necessary as a part of the core config mounting flow for ESLint.